build.gradle

```diff
@@ -0,0 +1,159 @@
+import java.text.SimpleDateFormat
+
+buildscript {
+    ext {
+        springBootVersion = '1.5.2.RELEASE'
+    }
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+    }
+    dependencies {
+        classpath 'io.github.robwin:jgitflow-gradle-plugin:0.5.0'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'findbugs'
+    id 'org.springframework.boot' version '1.5.9.RELEASE'
+    id 'jacoco'
+    id 'maven-publish'
+    id 'com.jfrog.artifactory' version '4.4.14'
+    id 'com.bmuschko.clover' version '2.2.0'
+}
+
+bootRepackage {
+    enabled = false
+}
+
+repositories {
+    mavenLocal()
```

Comment:
This line ensures gradle to check one's local repository before reaching out to the external repository.